src/etc/inc/interfaces.inc

```diff
@@ -4357,7 +4357,7 @@ function get_interfaces_info()
         if ($bridge) {
             $bridge_text = `/sbin/ifconfig {$bridge}`;
             if (stristr($bridge_text, "blocking") <> false) {
-                $ifinfo['bridge'] = "<b><font color='red'>" . gettext("blocking") . "</font></b> - " . gettext("check for ethernet loops");
+                $ifinfo['bridge'] = "<b><span style='color: red'>" . gettext("blocking") . "</span></b> - " . gettext("check for ethernet loops");
```

Comment:
how about this one in text-danger then? :D

Reply:
As you wish ;)